__tests__/verify-dafny.sh

```diff
@@ -8,9 +8,25 @@ if [ -z "$1" ]; then
   exit 1
 fi
 
-dafny_version="$(dafny /version)"
+input=$1
+version="-version"
+
+if [[ $input = nightly* ]]
+then
+  echo "Nightly versions do not output a version number"
```

Comment:
```suggestion
  echo "Nightly versions do not output a version number that matches the full version string"
```

Not having a version number at all was actually a bug in mainline that got fixed last week: https://github.com/dafny-lang/dafny/issues/2813